advanced_guides/binary.md

```diff
@@ -0,0 +1,70 @@
+# MeiliSearch Binary
+
+
+## How to compile meiliSearch
+
+### Dependencies 
+
+MeiliSearch is made in `rust`. Because of that `rust` and his package manager `cargo` must [be installed](https://www.rust-lang.org/tools/install) to compile the project.
```

Comment:
```suggestion
MeiliSearch is made in `rust`. Because of that `rust` and its package manager `cargo` must [be installed](https://www.rust-lang.org/tools/install) to compile the project.
```